CHANGELOG.unreleased.md

```diff
@@ -15,6 +15,7 @@
 
 - [LDAP] "Synchronize LDAP groups" button: fix imported LDAP users not being correctly added or removed from groups in some cases (PR [#5545](https://github.com/vatesfr/xen-orchestra/pull/5545))
 - [VM migration] Fix `VIF_NOT_IN_MAP` error (PR [5544](https://github.com/vatesfr/xen-orchestra/pull/5544))
+- [Task] The position of the search bar are next to the filter `item per page` (PR [#5584](https://github.com/vatesfr/xen-orchestra/pull/5584))
```

Comment:
Is it a fix?

Reply:
if we are considering the dropdown isn't at his good position, we can call that a bug fix no ?